libs/platform/unit/platform_unit_test.cpp

```diff
@@ -371,3 +375,82 @@ TEST_CASE("serialize_map_test", "[platform]")
     if (buffer != nullptr)
         free(buffer);
 }
+
+TEST_CASE("serialize_program_information_test", "[platform]")
+{
+    _test_helper test_helper;
+
+    ebpf_helper_function_prototype_t helper_prototype[] = {
+        {1000,
+         "helper_0",
+         EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+         {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
+        {1001,
+         "helper_1",
+         EBPF_RETURN_TYPE_INTEGER,
+         {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE}}};
+    ebpf_context_descriptor_t context_descriptor = {32, 0, 8, -1};
```

Comment:
Unless you're explicitly trying to enter invalid values for a case that should fail, don't hard code the size/offset numbers, use sizeof(), field offsets, etc

Reply:
Unresolving since code still has 32, 0, and 8 hard coded and I didn't see a response.  If the values are completely arbitrary and not used for anything in this test, add a // comment saying so.

Reply:
I thought I replied before resolving the thread. Not sure what happened. Yes, these are arbitrary values and have no consequence to the serialization process. Left a comment like you suggested.